RUTextSize/Classes/Utility/RUAttributesDictionaryBuilder.m

```diff
@@ -173,12 +186,39 @@ +(nonnull NSString*)attributeTypeKeyForEnum:(RUAttributesDictionaryBuilder_attri
 			break;
 
 		case RUAttributesDictionaryBuilder_attributeType_kerning:
-			return NSParagraphStyleAttributeName;
+			return NSKernAttributeName;
 			break;
 	}
 	
 	NSAssert(false, @"unhandled attributeType %li",attributeType);
 	return nil;
 }
 
+#if DEBUG
+#pragma mark - Unit Testing
++(void)DEBUG__RUAttributesDictionaryBuilder_RUTextSize_kerning_unitTest
+{
+	NSString* const superLongText = [NSString ru_exampleString_longestTest];
+
+	UILabel* const debugLabel = [UILabel new];
```

Comment:
No need to use a label for this test. The test should simply use `ru_textSizeWithBoundingWidth:` on `NSAttributedString`.